libs/langchain/langchain/schema/runnable.py

```diff
@@ -192,6 +222,146 @@ def _call_with_config(
             )
             return output
 
+    def _stream_with_config(
+        self,
+        input: Iterator[Input],
+        _transform: Callable[[Iterator[Input]], Iterator[Output]],
```

Comment:
should not be marked private in a function/method signature, `transformer` or `transform`

```suggestion
        transform: Callable[[Iterator[Input]], Iterator[Output]],
```